lib/manageiq/providers/scvmm/engine.rb

```diff
@@ -3,6 +3,10 @@ module Providers
     module Scvmm
       class Engine < ::Rails::Engine
         isolate_namespace ManageIQ::Providers::Scvmm
+
+        def self.plugin_name
+          _('Microsoft Provider')
```

Comment:
Since there are technically two Microsoft providers (Azure being the other one) I think `Microsoft SCVMM Provider` is more descriptive.

Reply:
All right, fixed.